cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,48 @@
-describe('', () => {
-  before(() => {
+describe('Conduit page', () => {
+  const faker = require('faker');
 
-  });
+  let user;
+
+  const article = {
+    title: faker.lorem.word(),
+    description: faker.lorem.word(),
+    body: faker.lorem.word()
+  };
 
-  it('', () => {
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
+    cy.visit('https://conduit.mate.academy/');
   });
 
-  it('', () => {
+  it('should provide the ability to create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+
+    cy.get('[placeholder="Article Title"]').type(article.title);
+    cy.get('[placeholder="What\'s this article about?"]')
+      .type(article.description);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(article.body);
+
+    cy.get('.btn').click();
```

Comment:
IMHO  better specify this btn like (cy.contains('button', 'Publish Article')

Reply:
thanks, I will use it in future tests:)